README.md

```diff
@@ -25,3 +25,15 @@ To run the tests:
 ```bash
 npm test
 ```
+
+## Test app
+
+This is very much a work in progress.
+
+To run the test app:
+
+```
+$ cd test/app
```

Comment:
I think app/ would imply that it was normal usage of it. But maybe testapp/ or something like that?